### PR TITLE
Feat/GitHub actions

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

**Review by JAdshead — DISMISSED** [review]

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '12.x', '14.x', '16.x' ]
```
Comment (JAdshead) [inline .github/workflows/tests.yml:13]: current test target 10, 12 and 14 so we should probably stick to that
```suggestion
        node: [ '10.x', '12.x', '14.x' ]
```

File: .github/workflows/health-check.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  schedule:
+    # At minute 0 past hour 0800 and 2000.
+    - cron:  '0 8,20 * * *'
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10.x', '12.x', '14.x' ]
```
Comment (10xLaCroixDrinker) [inline .github/workflows/health-check.yml]:
```suggestion
        node: [ '12.x', '14.x', '16.x' ]
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
```
Comment (10xLaCroixDrinker) [inline .github/workflows/release.yml]:
```suggestion
          node-version: 16
```

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10.x', '12.x', '14.x' ]
```
Comment (10xLaCroixDrinker) [inline .github/workflows/tests.yml]:
```suggestion
        node: [ '12.x', '14.x', '16.x' ]
```

**Review by 10xLaCroixDrinker — CHANGES_REQUESTED** [review]

**Review by 10xLaCroixDrinker — APPROVED** [review]

**Review by eddhurst — APPROVED** [review]

**Pull request merged**